src/test/java/bio/terra/app/controller/StatusTest.java

```diff
@@ -21,13 +25,19 @@
 import org.springframework.test.web.servlet.MvcResult;
 
 @RunWith(SpringRunner.class)
-@SpringBootTest
+@SpringBootTest(properties = {"tdr.testNoDatabase=true"})
 @AutoConfigureMockMvc
 @Category(Unit.class)
 public class StatusTest {
 
   @Autowired private MockMvc mvc;
   @Autowired private ConfigurationService configurationService;
+  @Autowired private DatasetDao datasetDao;
+
+  @Before
+  public void setup() throws Exception {
+    when(datasetDao.statusCheck()).thenReturn(new RepositoryStatusModelSystems().ok(true));
```

Comment:
This seems odd to me. The code is assuming that `datasetDao` is a mock; how does it know that this is always the case? Can't this be done in the code that's making the mock for this object?

Reply:
I've made it so that the beans can be explicitly mocked in each test rather than assuming they're mocked in the Spring context.